rhel/SPECS/dbaas-tools.spec

```diff
@@ -16,10 +20,13 @@ Summary:        A set of tools for Percona DBaaS
 License:        ASL 2.0
 URL:            https://github.com/kubernetes-sigs/aws-iam-authenticator
 Source0:        https://github.com/kubernetes-sigs/aws-iam-authenticator/archive/%{commit_aws}/aws-iam-authenticator-%{shortcommit_aws}.tar.gz
```

Comment:
Please add a comment explaining why download by SHA-1 hash, not by tag name (because git tag can be moved)

Reply:
Added in bc0f496